_posts/2015/2015-11-16-carbon-automobiles-bebop-fashion.md

```diff
@@ -0,0 +1,148 @@
+
+George B. Selden was awarded a patent in 1985 for the “Road Vehicle”, which is essentially a horse carriage powered by a gasoline combustion engine. Note that this was not the first gasoline combustion engine (which was created in 1859) or even the first “automobile”. The idea of the automobile had existed for most of that century. But the Selden patent was used to monopolize the industry and force manufacturers to adhere to their demands or risk a lawsuit.
+
+![](/images/osos/osos.017.jpeg)
+
+In 1902, a young entrepreneur named Henry Ford attempted to secure a license from the Association of License Automotive Manufacturers, but was denied because of "past business failings”. Getting denied may have had more to do with the influence of an established competitor in Detroit named Fredrick Olds being on the board of the association.
+
+Ford kept making cars anyway and eventually the Association filed a suite against him. After a long and public battle, Ford won a challenge and the patent was overturned in 1911.
+
+> “It is perfectly safe to say that George Selden has never advanced the automobile industry in a single particular...and it would perhaps be further advanced than it is now if he had never been born.”
+> <cite>Henry Ford</cite>
+
+_Aside: I didn't realize Hacker News was a thing in 1911._
+
+Shortly after, Ford helped establish what eventually became know as the Motor Vehicle Manufacturers Association, which established a cross-licensing agreement among all US auto manufacturers. Patents were shared openly and without the exchange of money among all the manufacturers. Anyone could join the Association and gain access to the existing patents on the condition that they contributed all of their patents to the pool.
+
+Here are the number of automobiles produced in the US by Ford and other auto manufacturers. See if you can guess when the patent was overturned.
+
+![](/images/osos/osos.022.jpeg)
+
+Even in the next 15 years, despite World War I and the Great Depression, the automobile industry experienced explosive growth.
+
+![](/images/osos/osos.025.jpeg)
+
+> “The long-zoom approach lets us see that openness and connectivity may, in the end, be more valuable to innovation than purely competitive mechanisms.”
+> <cite>Steven Johnson</cite>
+
+Instead of expending their efforts on reinventing the wheel, early auto manufacturers could focus on creating an industry and the infrastructure to support it. Automobiles and carbon show us that openness creates a platform that raises society up and allows us all to focus on solving the next problem.
+
+This is what open source is doing in the software world. By licensing the building blocks of software systems for others to freely use, we are opting-in to the model of the early automobile industry, where anyone can use the best ideas. Instead of competing with each other, we grow an entirely new industry.
```

Comment:
"Instead of competing" is a bit woolly. Fierce competition in delivering more value to consumers, but opted out of rent seeking competition (well in the form of IP, probably not others).